### DISCUSSION TITLE
Missing `Trending Repositories` section

### DISCUSSION THREAD

**Discussion by TanmoySG** [community/community#50730] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

<img width="414" alt="image" src="https://user-images.githubusercontent.com/36238254/226984444-e23bc2ed-2e3b-4031-895e-af5f26e74012.png">

Is it just me or the `Trending Repositories` is missing from the first page/feed ?

**Comment by Neodevils** [comment]
Hello 👋🏻

If you are talking about Feeds, it's there.

![Screenshot_2023-03-22-20-30-14-741_com android chrome](https://user-images.githubusercontent.com/71941576/226989060-4be715b7-58b7-41ba-8685-3128be59cdc4.jpg)

**Reply by TanmoySG** [reply]
Ah this I too have but the Trending Repositories Section on the Right side of the Github Page just below the GitHub changelogs/announcements used to be noice!

**Reply by Neodevils** [reply]
Hey again! 

You can use this link to explore repository too:
https://github.com/explore

But I got what you meant. I'm just showing off other variants possible to satisfy you.

**Reply by TanmoySG** [reply]
Thanks @Neodevils !!!

**Reply by Neodevils** [reply]
> Thanks @Neodevils !!!

Please mark this discussion as solved and closed! All will be good! 🥳
